### PR TITLE
Updated Weather Metadata

### PR DESCRIPTION
**Pull request by allisterakun**
The weather section of the metadata only contains the `precip` variable and is missing the following variables:
- `year`
- `jday`
- `high`
- `low`
- `avg`
- `Hday`
- `irrigation`

This PR updates the weather section of all the metadata files to include the missing variables.

### CODE REVIEW

File: input/metadata/ARL_metadata.json [hunk 1]
```diff
@@ -2392,11 +2392,49 @@
       }
     },
     "weather_properties": {
+      "year": {
+        "description": "Year.",
+        "type": "number",
+        "minimum": 1900,
+        "default": 0
```
Comment (PooyaHekmati) [inline input/metadata/ARL_metadata.json]: why do we want to default the year to 0?

File: input/metadata/ARL_metadata.json [hunk 2]
```diff
@@ -2392,11 +2392,49 @@
...
+      },
+      "jday": {
+        "description": "Julian day of the year.",
+        "type": "number",
+        "minimum": 0,
+        "maximum": 366,
+        "default": 0
```
Comment (PooyaHekmati) [inline input/metadata/ARL_metadata.json]: Same as the year, why default to 0?

File: input/metadata/ARL_metadata.json [hunk 3]
```diff
@@ -2392,11 +2392,49 @@
...
-        "description": "amount of precipitation",
+        "description": "Amount of precipitation (mm H2O).",
         "type": "number",
         "minimum": 0,
         "default": 0
+      },
+      "high": {
+        "description": "The maximum air temperature of the day (°C).",
+        "type": "number",
+        "default": 0
```
Comment (PooyaHekmati) [inline input/metadata/ARL_metadata.json]: 0C doesn't sound like a high temperature :)

File: input/metadata/ARL_metadata.json [hunk 4]
```diff
@@ -2392,11 +2392,49 @@
...
+      },
+      "Hday": {
+        "description": "The solar radiation of the day (MJ m^-2).",
+        "type": "number",
+        "default": 0
+      },
+      "irrigation": {
+        "description": "Irrigation.",
```
Comment (PooyaHekmati) [inline input/metadata/ARL_metadata.json]: mm h2o?

**Review by PooyaHekmati — APPROVED** [review]

**Review by ew3361zh — APPROVED** [review]
LGTM, nice work, Allister!

**Pull request merged**